TypeScript 5 support?

Hi,
What's the best way to help with TypeScript 5 support as we're unable to upgrade our projects because of this module's dependency on 3 or 4.

`npm ERR! Could not resolve dependency:
npm ERR! peer typescript@"^3.7.x || ^4.3.x" from jest-html-reporter@3.7.0
npm ERR! node_modules/jest-html-reporter
npm ERR!   dev jest-html-reporter@"^3.7.0" from the root project`

Cheers 👍

--- a/jest.config.json
+++ b/jest.config.json
@@ -1,10 +1,13 @@
 {
   "roots": ["<rootDir>"],
   "preset": "ts-jest",
-  "globals": {
-    "ts-jest": {
-      "babelConfig": true
-    }
+  "transform": {
+    "^.+\\.tsx?$": [
+      "ts-jest",
+      {
+        "babelConfig": true
+      }
+    ]
   },
   "collectCoverage": true,
   "collectCoverageFrom": [

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "jest-html-reporter",
-  "version": "3.7.0",
+  "version": "3.7.1",
   "description": "Jest test results processor for generating a summary in HTML",
   "main": "dist/index.js",
   "unpkg": "dist/index.js",
@@ -51,7 +51,7 @@
   },
   "peerDependencies": {
     "jest": "19.x - 29.x",
-    "typescript": "^3.7.x || ^4.3.x"
+    "typescript": "^3.7.x || ^4.3.x || ^5.x"
   },
   "devDependencies": {
     "@rollup/plugin-babel": "^5.3.0",

